string agent
float64 time
ActionVisu[] actions
